crypto/box/src/hkdf_box.rs

```diff
@@ -75,10 +75,7 @@ where
         use mc_crypto_keys::KexPublic;
         let (our_public, shared_secret) = key.new_secret(rng);
 
-        // FIXME: using to_bytes API here seems to confuse the compiler
-        let curve_point_bytes = // our_public.to_bytes();
-            our_public.map_bytes(|bytes|
-            GenericArray::<u8, <KexAlgo::Public as ReprBytes>::Size>::clone_from_slice(bytes));
```

Comment:
whatever wasn't working here, it seems to be fixed since most recent compiler uprev